io/buffer_test.ts

```diff
@@ -357,7 +357,7 @@ Deno.test("bufferTestGrow", async () => {
   }
 });
 
-Deno.test("testBufferBytesArrayBufferLength", () => {
+Deno.test("writeAllSync() and readFromSync()", () => {
```

Comment:
```suggestion
Deno.test("Buffer.write() and Buffer.readFromSync()", () => {
```
Note: `writeAll()` is not a `Buffer` method.